docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3172,6 +3176,7 @@ Sets up an exec instance in a running container `id`
         container. Format is a single character `[a-Z]` or `ctrl-<value>`
         where `<value>` is one of: `a-z`, `@`, `^`, `[`, `,` or `_`.
 -   **Tty** - Boolean value to allocate a pseudo-TTY.
+-   **Env** - A list of environment variables in the form of `["VAR=value"[,"VAR2=value2"]]`
```

Comment:
Can you change the "form" example to:

```Markdown
`["VAR=value", ...]`
```

The other format was confusing some people (because they interpreted the square brackets as being part of the JSON)